sinonjs/samsam
# WS-2018-0590: Upgrade mocha and mochify to latest

This takes care of the insecure diff package

#### How to verify - mandatory

1. Check out this branch
2. `npm install`
3. `npm test`
1. Observe that tests pass

#### Checklist for author
